MobileBuy/buy/src/main/java/com/shopify/buy/utils/AndroidPayHelper.java

```diff
@@ -71,13 +72,16 @@
  */
 
 public final class AndroidPayHelper {
+
     private static final int FIRST_NAME_PART = 0;
     private static final int LAST_NAME_PART = 1;
 
     public static final int REQUEST_CODE_MASKED_WALLET = 500;
     public static final int REQUEST_CODE_CHANGE_MASKED_WALLET = 501;
     public static final int REQUEST_CODE_FULL_WALLET = 502;
 
+    public static final String[] UNSUPPORTED_COUNTRIES_FOR_SHIPPING = {"MM", "SS", "GG", "IM", "KP", "SX", "SY", "IR", "BL", "BQ", "SD", "CU", "CW", "AX", "MF", "JE"};
```

Comment:
Can you add a comment saying where these came from, ideally a link to documentation or something? I assume there is no way to derive this list.